django/core/jinja2/base.jinja

```diff
@@ -288,6 +288,21 @@
     <link rel="mask-icon" href="{{ static('favicon/safari-pinned-tab.svg') }}" color="#5bbad5">
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="theme-color" content="#ffffff">
+    {% block ogp_tags %}
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="https://www.comses.net/">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="CoMSES Net">
```

Comment:
should we include the full expanded acronym here, Network for Computational Modeling in the Social and Ecological Sciences or will it be too long for the rich preview?

Reply:
We could test it the full expanded acronym, I have changed it to the full thing for now. If it seems to long later I can change it back.